pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <artifactId>gatling-aws-maven-plugin</artifactId>
     <groupId>com.ea.gatling</groupId>
-    <version>1.0.18-SNAPSHOT</version>
+    <version>1.0.19-SNAPSHOT</version>
```

Comment:
Maven release plugin would publish 1.0.18 version and auto-increment SNAPSHOT version. Please exclude changes  to version from pull request